<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>251</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="Date">
    <item row="2" column="2">
     <widget class="QLineEdit" name="stringTimeEdit"/>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="stringDateEdit"/>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="stringDateTimeEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>日   期</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>时   间</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>日期时间</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTimeEdit" name="timeEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item row="4" column="1">
     <widget class="QDateTimeEdit" name="dateTimeEdit"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="Btn_GetTime">
      <property name="text">
       <string>读取当前日期时间</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>字符串显示</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="BtnSetTime">
      <property name="text">
       <string>设置时间</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="BtnSetDate">
      <property name="text">
       <string>设置日期</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="BtnSetDateTime">
      <property name="text">
       <string>设置日期时间</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>260</width>
     <height>135</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLCDNumber" name="lcdMinute"/>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="BtnSet">
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="TimeInterval"/>
    </item>
    <item row="1" column="0">
     <widget class="QLCDNumber" name="lcdHour"/>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QLCDNumber" name="lcdSecond"/>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="BtnStart">
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="BtnStop">
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="BtnReset">
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>定时周期(ms）</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>流逝时间</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="ElapsTime">
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
